### DISCUSSION TITLE
Original model

### DISCUSSION THREAD

**Discussion by Gao0130** [Schievel1/dactyl_manuform_r_track#9] [Q&A]
Can I have the original model? I already finished this, I think this model's Hot Swaps slot are really nice to use when I make this keybroad. Can I have one? thanks!!!

**Comment by Schievel1** [comment]
Sorry I don’t understand, what is it you’re asking for? The CAD files?

**Reply by Gao0130** [reply]
Yeah!!! CAD

**Reply by Gao0130** [reply]
which don't have tracking ball

**Reply by Schievel1** [reply]
> which don't have tracking ball 

I don't have the models without trackball. But I could give you the freecad files and you could just mirror the left side.

https://gitlab.com/keyboards1/dm_left

Also there is a generator which could give you the models you want. In my opinion the hot swap socket holder this generator has are better than the ones from this model here. https://github.com/Narex45635/dactyl-keyboard-python-btu-trackball
